tools/stern/src/Stern/API.hs

```diff
@@ -627,7 +660,7 @@ groupByStatus conns =
     byStatus s = length . filter ((==) s . ucStatus)
 
 ifNothing :: Error -> Maybe a -> Handler a
-ifNothing e = maybe (throwM e) return
+ifNothing e = maybe (throwE e) return
```

Comment:
So this throws errors in `ExceptT` instead of `IO` now? Good.

Reply:
Yes, I actually realised that `throwM` didn't quite work. It was throwing 5xx.